ipaserver/install/certs.py

```diff
@@ -165,6 +162,27 @@ def __init__(self, realm, nssdir, fstore=None,
     ca_subject = ipautil.dn_attribute_property('_ca_subject')
     subject_base = ipautil.dn_attribute_property('_subject_base')
 
+    # migration changes paths, just forward attribute lookup to nssdb
+    @property
```

Comment:
I remember doing NSS-related refactoring last year. I was promised CertDB and NSS would go away so I don't have to care about it that much. That however did not happen so an obvious question - if CertDB was properly inheriting from NSSDatabase, this would not be necessary, right?

Reply:
Subclassing isn't always the right solution. Encapsulation works fine if done correctly.